### DISCUSSION TITLE
Stuck on corrupt file

### DISCUSSION THREAD

**Discussion by jg1xmv** [WinMerge/winmerge#902] [Q&A]
Hi,

When trying to compare folders, some files in one of the folders are corrupt and WinMerge just hangs trying to read these files. Is there an option whereby WinMerge gives-up trying to read a file after a given amount of time and moves on to the next available file so that it does not get bogged down?

Thanks,
Arnaud

**Comment by sdottaka** [comment]
> Is there an option whereby WinMerge gives-up trying to read a file after a given amount of time and moves on to the next available file so that it does not get bogged down?

Unfortunately, WinMerge doesn't have such an option.

**Reply by jg1xmv** [reply]
Ok, understood, thank you for the reply.

Still a great tool, thank you to all the developers!
